sklearn/cross_decomposition/_pls.py

```diff
@@ -480,7 +480,7 @@ def predict(self, X, copy=True):
         Parameters
         ----------
         X : array-like of shape (n_samples, n_features)
-            Training vectors, where n_samples is the number of samples and
+            Test samples, where n_samples is the number of samples and
```

Comment:
This was updated on `main` to be more generic:

https://github.com/scikit-learn/scikit-learn/blob/4b53fc3f67fa6d7966bd51db7c9d754cd187d48f/sklearn/cross_decomposition/_pls.py#L369-L370